bchd.go

```diff
@@ -61,7 +62,7 @@ func bchdMain(serverChan chan<- *server) error {
 	defer bchdLog.Info("Shutdown complete")
 
 	// Show version at startup.
-	bchdLog.Infof("Version %s", version())
+	bchdLog.Infof("String %s", version.String())
```

Comment:
Shouldn't this still be "Version"?

Reply:
You are too fast. Yes. I also forgot to gofmt the test.